src/lib/inactivity.js

```diff
@@ -0,0 +1,16 @@
+export const inactivityTime = () => {
+  let time;
+  window.onload = resetTimer;
+  document.onmousemove = resetTimer;
+
+  function logout() {
+    window.location.href = "/";
+  }
+  function resetTimer() {
+    clearTimeout(time);
+    time = setTimeout(logout, 10000); // 10 seconds
+    // time = setTimeout(logout, 300000); // 5 minutes
+    console.log(time);
+  }
+};
+inactivityTime();
```

Comment:
Suggestion: you called function twice in the PR : in this script and in 1-batiment/js/script.js 
I think we shouldn't call the function automatically when the script is imported, but rather calling it manually only when we want to